java/client/build.gradle

```diff
@@ -39,11 +39,11 @@ dependencies {
 
 ext {
     checkProtocVersion = { String output ->
-        // Line in format like: libprotoc 26.1
+        // Line in format like: libprotoc 28.1
         int majorVersion = Integer.parseInt(output.split(" ")[1].split("\\.")[0].trim());
         int minorVersion = Integer.parseInt(output.split(" ")[1].split("\\.")[1].trim());
-        if (majorVersion < 26 || (majorVersion == 26 && minorVersion < 1)) {
-            throw new GradleException("Protobuf compiler (protoc) version 26.1 or newer is required. Current version: $output");
+        if (majorVersion < 28)
```

Comment:
Runtime of 28.1 is compatible with code generated by version 28.0, but not vice-versa